core/services/pg/q.go

```diff
@@ -279,7 +282,14 @@ func (q Q) postSqlLog(ctx context.Context, begin time.Time) {
 	if ctx.Err() != nil {
 		q.logger.Debugf("SQL CONTEXT CANCELLED: %d ms, err=%v", elapsed.Milliseconds(), ctx.Err())
 	}
-	if slowSqlThreshold > 0 && elapsed > slowSqlThreshold {
+
+	slowThreshold := DefaultQueryTimeout
+	if q.QueryTimeout > 0 {
+		slowThreshold = q.QueryTimeout
+	}
+	// The idea is to always use the `SLOW_SQL_THRESHOLD` ENV var but the 1/10th of whatever the timeout value has been set to.
+	slowThreshold = slowThreshold / 10
+	if slowThreshold > 0 && elapsed > slowThreshold {
```

Comment:
Looks like this comment needs updating. Or what about something like this instead?
```suggestion
	timeout := q.QueryTimeout
	if timeout <= 0 {
		timeout = DefaultQueryTimeout
	}
	slowThreshold := timeout / 10
	if slowThreshold > 0 && elapsed > slowThreshold {
```